src/pykeen/experiments/literale/kristiadi2019_complex+literale_glin_fb15k237.yaml

```diff
@@ -0,0 +1,32 @@
+metadata:
+  title: "Learn FB15k-237 Dataset with ComplEx+LiteralE-g_lin as described by Kristiadi et al., 2019"
+pipeline:
+  dataset: FB15k237WithLiterals
+  dataset_kwargs:
+    create_inverse_triples: true
+    numeric_literals_preprocessing: "minmax"
+    numeric_triples_preprocessing: "filter_by_relations"
+    force: true # necessary for applying the above preprocessing functions in case a previous experiment with the same dataset has been performed
```

Comment:
Why is this necessary? If the `dataset_kwargs` change, the hash should change, too.

cf. https://github.com/pykeen/pykeen/blob/b3ef9d19883967b41f7fdc2b540972ebcb35f273/src/pykeen/datasets/utils.py#L210